packages/cms-lib/fileManager.js

```diff
@@ -51,6 +58,77 @@ async function uploadFolder(portalId, src, dest, { cwd }) {
   }
 }
 
+/**
+ *
+ * @param {number} portalId
+ * @param {object} file
+ * @param {string} dest
+ * @param {string} folderPath
+ */
+async function fetchFile(portalId, file, dest, folderPath) {
+  const relativePath = `${folderPath}/${file.name}.${file.extension}`;
+  const destPath = convertToUnixPath(path.join(dest, relativePath));
```

Comment:
You'll want to make sure to always use `path.join` when working with paths for the local file system as well as https://github.com/HubSpot/hubspot-cms-tools/blob/master/packages/cms-lib/path.js#L14-L23 so that everything will work as expected on windows.